WIP - OADP-3522: ROSA docs is missing instructions related to the internal … (#72112)
openshift/openshift-docs: modules/installing-oadp-rosa-sts.adoc

        enable: false
EOF
----

> <1> Internal image backup is supported with ROSA. Only set this field to false, you want to explicitly skip the image backup.

Reviewer: ```suggestion
<1> ROSA supports internal image backup. Set this field to `false` if you do not want to use image backup.
```

    name: ${CLUSTER_NAME}-dpa
    namespace: openshift-adp
  spec:

>     backupImages: true

Reviewer 2: Do we need one of these here also?

> <1> ROSA supports internal image backup. Set this field to `false` if you do not want to use image backup.

Author: Nice!
